doc/source/serve/production-guide/index.md

```diff
@@ -0,0 +1,119 @@
+(serve-in-production)=
+
+# Production Guide
+
+The recommended way to run Ray Serve in production is on Kubernetes using the [KubeRay](kuberay-quickstart) [RayService] custom resource.
+The RayService custom resource automatically handles important production requirements such as health checking, status reporting, failure recovery, and upgrades.
```

Comment:
It's splitting hairs, but
`RayService custom resource` -> `RayService controller`